### ISSUE TITLE
rtfobj - ole2link parsing error

### ISSUE THREAD

**Issue by securitykitten** [decalage2/oletools#178]
When looking at the RTF (fe2e5d0543b4c8769e401ec216d78a5a3547dfd426fd47e097df04a5f7d6d206), it would appear that rtfobj is getting thrown off by some malformed data.

```
THIS IS WORK IN PROGRESS - Check updates regularly!
Please report any issue at https://github.com/decalage2/oletools/issues

===============================================================================
File: 'fe2e5d0543b4c8769e401ec216d78a5a3547dfd426fd47e097df04a5f7d6d206' - size: 6215 bytes
---+----------+-------------------------------+-------------------------------
id |index     |OLE Object                     |OLE Package                    
---+----------+-------------------------------+-------------------------------
0  |000002E0h |Not a well-formed OLE object   |                               
---+----------+-------------------------------+-------------------------------
```
The parser seems to get tripped up around the attribute placed in this section of code

```
ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff0100000208000000010000000000000000000000000000005c010000e0c9ea79f9bace118c8200aa004b{\*\private\b a90b4401000048007400540050003}
a005C005C00380034002e003200300030002e00310036002e003200340032002f006d0079006700750079002e0078006c007300000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
```
When taking the first couple bytes of the stream after the malformed tag, it becomes clear the intent of this object

```
echo "5C005C00380034002e003200300030002e00310036002e003200340032002f006d0079006700750079002e0078006c0073" | sed 's/00//g' | xxd -r -p
\\84.200.16.242/myguy.xls
```

A copy of the RTF can be found here https://drive.google.com/file/d/0B6sLf2D8GZIQMFFnTDJzTm9KUTQ/view?usp=sharing

Password: infected

**Comment by decalage2** [comment]
This bug is fixed in the latest dev version (commit 3ec4b06) and the sample RTF can be analyzed properly. However, maybe I will need to check if other RTF control words need to be ignored like "\private". (ref: RTF specifications v1.9.1 page 47  [[MSFT-RTF]](http://go.microsoft.com/fwlink/?LinkId=120924))

**Comment by decalage2** [comment]
This specific issue has been fixed, I created issue #181 to check other RTF control words.
